src/aks-preview/azcli-aks-live-test/transcribe_env.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# clear
+cat /dev/null > env.list
+
+# tenant, sub, client
+echo "TENANT_ID=$TENANT_ID" >> env.list
+echo "AZCLI_ALT_SUBSCRIPTION_ID=$AZCLI_ALT_SUBSCRIPTION_ID" >> env.list
+echo "AZCLI_ALT_CLIENT_ID=$AZCLI_ALT_CLIENT_ID" >> env.list
+echo "AZCLI_ALT_CLIENT_SECRET=$MAPPED_AZCLI_ALT_CLIENT_SECRET" >> env.list
```

Comment:
why do we need to store the secret?

Reply:
Write down the variables defined in pipeline to a file and pass this file to docker through "--env-file" option as environment variables. Maybe a better way to achieve this?

Reply:
I think we should not persist the secret in pipeline if possible. maybe we can just pass it through the environment variable.

Reply:
sure, updated in commit [1d4ba6b](https://github.com/Azure/azure-cli-extensions/pull/3292/commits/1d4ba6bcef6656355cd5ec980b82259ba820b37b).